config/config-template-long.yaml

```diff
@@ -96,19 +96,23 @@ scanners:
       format: ["json"]
       # format: ["json","html","sarif"]  # default: "json" only
 
+    urls:
+      # Optional, `includes` and `excludes` take a list of regexps.
+      # includes: A URL matching that regexp will be in the scope of scanning, in addition to application.url which is already in scope
+      # excludes: A URL matching that regexp will NOT be in the scope of scanning
+      # Note: The regular expressions MUST match the whole URL.
+      #       e.g.: 'http://example.com/do-not-descend-here/' will actually descend
+
+      #includes:
+      #  - "^https?://example.com:3000/do-not-descend-here/.*$"
```

Comment:
the "do-not-descend-here" in the URL feels like a counter intuitive example for the "includes", maybe renaming "descend-here" ?